src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -59,6 +59,7 @@ def is_stopping(self):
         return self.stopping.isSet()
 
     def shutdown(self):
+        # Overrides CephfsClient.shutdown()
```

Comment:
Not sure if `purge_queue.cancel_all_jobs()` is order dependent of the other calls, so opted for everride.

Reply:
that's fine